License.md

```diff
@@ -32,8 +32,8 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
-=== Perl Artistic License (version 2)
+## Perl Artistic License
 See the file docs/artistic.txt in the main distribution.
 
-=== GNU GPL version 2
+## GNU GPL version 2
```

Comment:
For this and the above, please make sure you add one space _after_ the ATX headers.